prime-router/src/main/kotlin/history/DeliveryHistory.kt

```diff
@@ -83,13 +81,7 @@ class DeliveryHistory(
      */
     val fileName: String
         get() {
-            return Report.formExternalFilename(
-                this.bodyUrl,
-                ReportId.fromString(this.reportId),
-                this.schemaName,
-                Report.Format.safeValueOf(this.bodyFormat),
-                this.createdAt
-            )
+            return this.bodyUrl ?: ""
```

Comment:
This doesn't seem quite right?  Doesn't the frontend show the something like "report.hl7"?

Reply:
updated to get filename from bodyUrl